packages/calling-component-bindings/src/videoGallerySelector.ts

```diff
@@ -147,7 +157,11 @@ export const videoGallerySelector: VideoGallerySelector = createSelector(
       ),
       dominantSpeakers: dominantSpeakerIds,
       /* @conditional-compile-remove(optimal-video-count) */
-      maxRemoteVideoStreams: optimalVideoCount
+      maxRemoteVideoStreams: optimalVideoCount,
+      /* @conditional-compile-remove(spotlight) */
+      spotlightedParticipants: spotlightedParticipants
```

Comment:
this should be memoized or every render pass it will return a new object

Reply:
Thanks for spotting that. Fixed.